Floofbot/Modules/Fun.cs

```diff
@@ -24,10 +24,7 @@ public class Fun : ModuleBase<SocketCommandContext>
             ".jpg", ".gif", ".png"
         };
 
-        [Command("8ball")]
-        [Summary("Ask the Magic 8-Ball a question")]
-        public async Task AskEightBall([Summary("question")][Remainder] string question)
-        {
+        public static string EightBallResponse() {
```

Comment:
Since the 8-ball is being called at multiple places, we might want to place a new 8-ball class under Modules/Helpers and initialize the eight ball whenever we need it, instead.

Reply:
I'm not sure how C# internals go, but I wouldn't like to instantiate a new object every time we send an EightBall response. Sounds like a way to lose memory and/or over-stress the garbage collector.

Reply:
True. You can make it a class with just static members then to keep things organized, since otherwise the bot is calling directly into this module sometimes, which should only be called by valid commands.

Reply:
Created a static helper class